scripts/language_model/data/classification.py

```diff
@@ -0,0 +1,349 @@
+# Copyright 2018 The Google AI Language Team Authors and DMLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""BERT classification/regression datasets."""
+
+
+__all__ = [
```

Comment:
Did you make any changes to this file? If not, how about just symlinking it to the version in `bert`?